src/components/Calendar/doubleCalendar/index.js

```diff
@@ -122,15 +129,22 @@ export default function DoubleCalendar(props) {
     );
 
     useEffect(() => {
-        setFocusedDate(normalizeDate(value));
-    }, [value]);
+        setFocusedDate(currentValue);
+        if (
+            !isSameMonth(currentValue, rightCalendarMonth) &&
+            !isSameMonth(currentValue, currentMonth)
+        ) {
+            setCurrentMonth(getFirstDayMonth(currentValue));
+        }
```

Comment:
this is the same logic as in useHandleKeyDown I think here you should do the useEffect.

Reply:
this logic is only to update focused date when value is passed via prop... is has nothing to do with key handler